backend/module/eCampCore/src/EntityService/CampCollaborationService.php

```diff
@@ -196,6 +197,25 @@ protected function deleteEntity(BaseEntity $entity): void {
             parent::deleteEntity($entity);
         } else {
             $this->patchEntity($campCollaboration, (object) ['status' => CampCollaboration::STATUS_INACTIVE]);
```

Comment:
alternativ, `$this->patch(...)` verwenden. Dann wird ACL auf Patch geprüft und validateEntity automatisch aufgerufen.

Reply:
Dann brauchst du die DELETE und die PATCH Berechtigung, um einen Benutzer auf Inaktiv zu setzen, plus du musst wieder von aussen in die Klasse (Entity mit der id fetchen usw).
Tönt für mich nicht sinnvoller.